### PR TITLE
Selection criteria

### PR DESCRIPTION
**Pull request by yavorska-iryna**
one more small edit to make sure that experience levels do not contain none

### CODE REVIEW

File: brain_observatory_analysis/dev/data_selection_tools.py [hunk 1]
```diff
@@ -35,7 +30,10 @@ def dateformat(exp_date):
     """
     from datetime import datetime
     if exp_date is not str:
-        exp_date = str(exp_date)[:-3]  # remove 3 zeros from seconds, otherwise the str is too long
+        if len(str(exp_date)) == 26:
+            exp_date = str(exp_date)[:-3]  # remove 3 zeros from seconds, otherwise the str is too long
+        else:
```
Comment (jkim0731) [inline brain_observatory_analysis/dev/data_selection_tools.py:35]: Why did you need this?

**Review by jkim0731 — CHANGES_REQUESTED** [review]
It was good, straightforward, just need a comment on the exp_date

File: brain_observatory_analysis/dev/data_selection_tools.py (hunk 1 above)
Comment (yavorska-iryna) [reply brain_observatory_analysis/dev/data_selection_tools.py:35]: if I recall correctly, not all exp dates are the same. So for the ones that are 26 characters, this would shorten it.

File: brain_observatory_analysis/utilities/image_utils.py [hunk 2]
```diff
@@ -136,3 +136,22 @@ def save_figure(fig, figsize, save_dir, folder, fig_title, formats=['.png', '.pd
...
+        Number of colors to generate
+    Returns
+    -------
+    list
+        List of RGB tuples
+    """
+
+    colors = []
+    for i in range(num_colors):
+        hue = i / num_colors
```
Comment (jkim0731) [inline brain_observatory_analysis/utilities/image_utils.py]: This does not reach 1 - is it intended?
And is this about grayscale or colors? (hsv input only has hue, no saturation and value)

**Review by jkim0731 — COMMENTED** [review]
Just one comment on generate_distinct_colors function

File: brain_observatory_analysis/utilities/image_utils.py (hunk 2 above)
Comment (yavorska-iryna) [reply brain_observatory_analysis/utilities/image_utils.py]: yes, this is because 1 would be white which is not visible, so it goes from light grey to black.

**Review by jkim0731 — APPROVED** [review]
All looks good!
